xml/deployment_scenarios.xml

```diff
@@ -0,0 +1,888 @@
+    <para>
+     To update the repository, you simply use <command>helm mirror</command> to
+     refresh the chart definitions on the server.
+    </para>
+    <procedure>
+     <step>
+      <para>
+      Install the
+      <link xlink:href="https://github.com/openSUSE/helm-mirror">helm-mirror</link>
+      plugin from the <filename>Containers</filename> module.
+     </para>
+<screen>&prompt.sudo;<command>zypper in helm-mirror</command>
+      </screen>
+      </step>
+      <step>
+       <para>
+      Download all charts from the repository.
+     </para>
+<screen>&prompt.user;<command>helm mirror https://kubernetes-charts.suse.com <replaceable>/charts/</replaceable></command>
+      </screen>
+     </step>
+     <step>
+      <para>
+       Transfer the contents of <filename>/charts/</filename> to the webserver
+       and unpack it to the desired location. The <filename>index.yaml</filename> file
+       must be in the root directory for your webserver.
+      </para>
+     </step>
+     <step>
+      <para>
+       Update the repository information on the &productname; admin node.
+      </para>
+<screen>&prompt.user;<command>helm repo update</command>
+       </screen>
+     </step>
+    </procedure>
+    <important>
+     <title>Refresh Container Images</title>
+     <para>
+      To use the mirrored Helm charts, you must also mirror the referenced
+      container images.
+     </para>
+      <para>
+       Refer to <xref linkend="sec.deploy.scenarios.airgap.container-registry.update"/>.
+     </para>
+    </important>
+   </sect3>
+  </sect2>
+
+  <sect2 xml:id="sec.deploy.scenarios.airgap.container-registry">
+   <title>Container Registry Mirror</title>
+   <para>
+    Container image registries are provided by SUSE, Docker, and other sources.
+    The &suse; container registry is used to update the &productname;
+    components.
+   </para>
+   <para>
+    Once configured, nothing has to be changed inside of Dockerfile(s),
+    Kubernetes manifest files, Helm charts or custom scripts, etc. All images
+    with a prefix <literal>registry.suse.com/</literal> will be automatically
+    pulled from the internal mirror.
+   </para>
+   <para>
+    Images that are located on other registries must be mirrored separately on
+    the airgapped network.
+   </para>
+
+   <sect3 xml:id="sec.deploy.scenarios.airgap.container-registry.configuration">
+    <title>Mirror Configuration</title>
+    <important>
+     <title>This example does not provide any security measures</title>
+     <para>
+      The default configuration is not using certificates to secure the
+      communication between this registry and all the nodes communicating with
+      it. There is also no authentication and authorization in place.
+     </para>
+     <para>
+      For production usage, we highly recommend deploying the registry using
+      TLS certificates and implementing authentication and authorization by
+      deploying a Portus instance.
+     </para>
```

Comment:
I think this can be dropped now.

@chentex proved we do not need to deploy a pull-through cache registry, we can deploy a regular one that uses basic http authentication.

Please rewrite this section with the help of @chentex